File: sum_square_difference/php/sum_square_difference.php
<?php
function sumSquares($max) {
	$total=0;
	for ($i=1; $i<=$max; $i++) {
		$total += $i**2;
	}
	return $total;
}

function squaresSum($max) {
	$total=0;
	for ($i=1; $i<=$max; $i++) {
		$total += $i;
	}
	return $total**2;
}

echo squaresSum(100) - sumSquares(100);

File: sum_square_difference/java/sum_square_difference.java
public class sum_square_difference {
	public static void main(String[] args) {
		System.out.println(squaresSum(100) - sumSquares(100));
	}

	public static int sumSquares(int max) {
		int total=0;
		for (int i=1; i<=max; i++) {
			total += Math.pow(i,2);
		}
		return total;
	}

	public static int squaresSum(int max) {
		int total=0;
		for (int i=1; i<=max; i++) {
			total += i;
		}
		return (int)Math.pow(total, 2);
	}
}
